### PR TITLE
Adding Javascript form validation.

### PR DESCRIPTION
**Pull request by saadyousafarbi**
The following input fields have been targeted to be validated using Javascript on the ```registration``` page:

* The entered ```email``` would be checked by use of regex,
* The entered ```password``` would be checked for its length (minimum length of 8).

All this is done on the ```register.html``` page containing registration functionality.

### CODE REVIEW

File: templates/register.html [hunk 1]
```diff
@@ -20,6 +20,22 @@
         }
       });
     });
+
+
+    $("#id_email").change(function() {
+        var email = $(this).val();
+        var email_pattern = /^\w+@[a-zA-Z_]+?\.[a-zA-Z]{2,3}$/;
+        if (!email_pattern.test(email)) {
+            alert("Incorrect Email.");
```
Comment (zubair-arbi) [inline templates/register.html]: Invalid email address.

File: templates/register.html [hunk 2]
```diff
@@ -20,6 +20,22 @@
...
+        }
+      });
+
+    $("#id_password").change(function() {
+        var password = $(this).val();
+        if (password.length < 8) {
+            alert("Password length is too short.");
```
Comment (zubair-arbi) [inline templates/register.html]: Password length should be at least 8 characters.

File: templates/register.html [hunk 3]
```diff
@@ -20,6 +20,22 @@
...
+        var email_pattern = /^\w+@[a-zA-Z_]+?\.[a-zA-Z]{2,3}$/;
+        if (!email_pattern.test(email)) {
+            alert("Invalid email address.");
+        }
+      });
+
+    $("#id_password").change(function() {
+        var password = $(this).val();
+        if (password.length <= 8) {
+            alert("Password length is too short.");
```
Comment (zubair-arbi) [inline templates/register.html]: what is the required minimum length then? You missed my suggestion.

Comment (saadyousafarbi) [reply templates/register.html]: ok

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Comment by saadyousafarbi** [comment]
Rebased b03c5ab87c195ad99c6e4acf7beadccd7800e942

**Pull request merged**